docs/resources/logdna_view.md

```diff
@@ -86,9 +87,9 @@ _Note:_ A `name` and at least one of the following properties: `apps`, `hosts`,
 `email_channel` supports the following arguments:
 
 - `emails`: _(Required)_ An array of email addresses (each email is of type _string_) to notify in the Alert
-- `immediate`: _(Optional)_ Whether the Alert will trigger immediately after the trigger limit is reached. Valid options are `true` and `false`, type _string_ (**Default: "false"**)
+- `immediate`: _(Optional)_ Whether the Alert will trigger immediately after the trigger limit is reached. _Note: Immediate can only be set to `true` for presence alerts_. Valid options are `true` and `false`, type _string_ (**Default: "false"**)
 - `operator`: _(Optional)_ Whether the Alert will trigger on the presence or absence of logs. Valid options are `presence` and `absence`, type _string_ (**Default: "presence"**)
-- `terminal`: _(Optional)_ Whether the Alert will trigger after the `triggerinterval` if the Alert condition is met (e.g., send an Alert after 30s). Valid options are `true` and `false`, type _string_
+- `terminal`: _(Optional)_ Whether the Alert will trigger after the `triggerinterval` if the Alert condition is met (e.g., send an Alert after 30s). Valid options are `true` and `false`, type _string_ (**Default: "true"**)
```

Comment:
For absence alerts are only terminal. Another way to say it is 
Absence alert - Terminal always true. Immediate always false.
Presence alert - Terminal can be true/false. Immediate can be true/false.